app/main.development.js

```diff
@@ -55,12 +59,66 @@ if (argv.version) {
   app.exit(0);
 }
 
-// Check if network was set on command line (but only allow one!).
+let rpcOptionsCount = 0;
+rpcOptionsCount += argv.rpcuser ? 1 : 0;
+rpcOptionsCount += argv.rpcpass ? 1 : 0;
+rpcOptionsCount += argv.rpccert ? 1 : 0;
+rpcOptionsCount += argv.rpcconnect ? 1 : 0;
+
+// Allow at most one network to be specified
 if (argv.testnet && argv.mainnet) {
-  logger.log(BOTH_CONNECTION_ERR_MESSAGE);
+  console.log(BOTH_CONNECTION_ERR_MESSAGE);
+  app.quit();
+} else if (!argv.spv && argv.spvconnect !== undefined) {
+  console.log(SPV_CONNECT_WITHOUT_SPV);
+  app.quit();
+} else if (argv.spv && argv.advanced) {
+  console.log(SPV_WITH_ADVANCED_MODE);
+  app.quit();
+} else if (!argv.advanced && (argv.rpcuser || argv.rpcpass || argv.rpccert || argv.rpcconnect)) {
+  console.log(RPC_WITHOUT_ADVANCED_MODE);
+  app.quit();
+} else if (rpcOptionsCount > 0 && rpcOptionsCount < 4) {
+  console.log(RPC_MISSING_OPTIONS);
   app.quit();
 }
 
+// Signal to renderer process that CLI options should override the global config
+if (argv.testnet) {
+  cliOptions.network = "testnet";
+} else if (argv.mainnet) {
+  cliOptions.network = "mainnet";
+}
+if (argv.advanced) {
+  cliOptions.daemonStartAdvanced = true;
+}
+if (argv.spv) {
+  cliOptions.spvMode = true;
+}
+if (argv.spvconnect !== undefined && isString(argv.spvconnect)) {
```

Comment:
Isn't just `isString()` sufficient for these (given that `isString(undefined) === false`)?

Reply:
True, the old code had both checks and I just used that.